lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -634,23 +633,6 @@ func getNetworkProbeConfig(ctx context.Context, networkID string) ([]*lte_mconfi
 	return npTasks, liUes
 }
 
-func getNetworkSentryConfig(network *configurator.Network) *lte_mconfig.SentryConfig {
```

Comment:
Eh tbh I would just leave this duplicated (since the return types are technically different), and add a note in this one that it's a dupe of the orc8r one